en/starter/faq.md

```diff
@@ -90,4 +90,11 @@ If you have a specific file, use the `res.sendFile()` function.
 If you are serving many assets from a directory, use the `express.static()`
 middleware function.
 
+<a name="which-is-the-minimum-version-of-nodejs-that-express-supports"></a>
+
+## Which is the minimum version of Node.js that Express supports?
```

Comment:
"What version of Node.js does Express require?"

Elsewhere you use "require" instead of "support", which is preferable IMO.

Reply:
Thanks @crandmck ! I'm not a native English speaker, so I welcome these language suggestions.

Change added [here](https://github.com/expressjs/expressjs.com/pull/1495/commits/db9404ed37c7de2f1dbf60dafbdeecf2c4acce52).